waku/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -257,6 +266,12 @@ proc getMessagesArbitraryQuery(s: PostgresDriver,
     for t in contentTopic:
       args.add(t)
 
+  if hexHashes.len > 0:
+    let cstmt = "messageHash IN (" & "?".repeat(hexHashes.len).join(",") & ")"
```

Comment:
Do we have an index built on this column? Would it be necessary for querying on message hash to be efficient? Perhaps we'll need a benchmark on a rather large DB (future work).

Reply:
I can't find any indexing.

Yes testing will tell us what to expect performance wise. We may have to use LMDB in the end (I hope not).